src/api/create.cpp

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2019, QuantStack and Mamba Contributors
+//
+// Distributed under the terms of the BSD 3-Clause License.
+//
+// The full license is in the file LICENSE, distributed with this software.
+
+#include "mamba/api/configuration.hpp"
+#include "mamba/api/install.hpp"
+
+
+namespace mamba
+{
+    void create()
+    {
+        auto& config = Configuration::instance();
+
+        config.load(!MAMBA_ALLOW_ROOT_PREFIX & !MAMBA_ALLOW_FALLBACK_PREFIX
```

Comment:
what does the `!MAMBA_ALLOW_ROOT_PREFIX & ... ` syntax achieve? Does that work? For inverting bits it's usually `~....` if I am not mistaken.

Reply:
!... should cast to bool, and then will likely return `false` so that should be `0 & 0 & 0` here? Or am I missing something?

Reply:
oO I should check.
I could have used `0` instead of this, but it looked more informative to understand what options are set to false instead

Reply:
I added some tests to check it works as expected. Negation of int just gives the information that the flag is not set.
```
EXPECT_EQ(!MAMBA_EXPECT_EXISTING_PREFIX, 0);
EXPECT_EQ((!MAMBA_ALLOW_ROOT_PREFIX) | MAMBA_ALLOW_EXISTING_PREFIX, MAMBA_ALLOW_EXISTING_PREFIX);
```

Reply:
I can remove that and use 0 instead, but I think it's much less informative

Reply:
I added a `MAMBA_NOT_...` for each possible check, that is a bit verbose but easier to read!